src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,139 @@
+import styled from "styled-components";
+
+const BASE_SRC = "assets/images";
+const BASE_HREF = "https://www";
+
+const logoMap = [
+  {
+    name: "facebook",
+    src: `${BASE_SRC}/facebook.svg`,
+    href: `${BASE_HREF}.facebook.com`,
+  },
+  {
+    name: "twitter",
+    src: `${BASE_SRC}/twitter.svg`,
+    href: `${BASE_HREF}.twitter.com`,
+  },
+  {
+    name: "youtube",
+    src: `${BASE_SRC}/youtube.svg`,
+    href: `${BASE_HREF}.youtube.com`,
+  },
+  {
+    name: "instagram",
+    src: `${BASE_SRC}/instagram.svg`,
+    href: `${BASE_HREF}.instagram.com`,
+  },
+];
+
+const Footer = () => {
+  return (
+    <footer>
+      <SFooterWrapper>
+        <SBottomNav>
+          <SCopyRight id="copyright">codeit - 2023</SCopyRight>
+          <div id="internal-links">
+            <div>
+              <a href="/privacy.html">Privacy Policy</a>
+              <a id="faq-link" href="/faq.html">
+                FAQ
+              </a>
+            </div>
+          </div>
+          <div id="external-links">
+            <SIconBox>
+              {logoMap.map((item) => {
```

Comment:
```suggestion
              {logoMap.map(({name, href, src}) => {
```
(참고만) 파라미터 선언 부분에서 destructuring을 활용하는건 어떨까요?